packages/webamp-modern-2/src/maki/interpreter.ts

```diff
@@ -311,6 +312,15 @@ class Interpreter {
             methodArgs.push(a.value);
           }
           const obj = this.stack.pop();
+          if (
+            !obj.value /* ==null */ &&
+            ((klass.name || "").toLowerCase() == "winampconfig" ||
+              (klass.name || "").toLowerCase() == "winampconfiggroup" ||
+              (klass.name || "").toLowerCase() == "configclass" ||
+              (klass.name || "").toLowerCase() == "config")
```

Comment:
This is definitely something we should clean up. Could you just add a comment explaining that this is a workaround?

Reply:
sure